package.json

```diff
@@ -33,5 +37,7 @@
     "broccoli-test-builder": "^0.2.0",
     "testem": "^0.9.0-1"
   },
-  "dependencies": {}
+  "dependencies": {
+    "broccoli-funnel": "^1.0.0"
```

Comment:
The dependencies are unfortunate for webpack users (like Romper), since they pull in things like `broccoli-funnel` to the browser as deps. Can this be done without adding broccoli funnel?